meteor/client/styles/tv2/keyboardPreview.scss

```diff
@@ -72,7 +68,7 @@
 	  }
 
 	  > .keyboard-preview__key__label {
-		font-size: 16px;
+		font-size: 0.59em;
```

Comment:
Don't know the exact difference between `em` and `rem`, but shouldn't this be in `rem`?

Reply:
Finsihed the review.
Now i know the difference of `em` and `rem`.